servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -32,11 +36,24 @@ def get_job_aggregations(timeFrame, projectId=None):
     job_name_summary = {}
     label_summaries = {}
 
-    for job in jobs:
-        _count_total_summary(job, total_summary)
-        _count_for_key(job, user_summary, job.extensions.user_id)
-        _count_for_key(job, job_name_summary, job.name)
-        _count_top_labels(job, label_summaries)
+    try:
+        for job in jobs:
+
+            # TEST_TOKEN_VALIE is a global config value used to distinguish testing jobs from batch to batch by timestemp
+            if 'TEST_TOKEN_VALUE' in current_app.config and (
+                    'test_token' not in job.labels or job.labels['test_token']
```

Comment:
Don't really need this key check, can just do: `or job.labels.get('test_token') != ...`